src/login/LoginPage.jsx

```diff
@@ -197,6 +279,7 @@ const LoginPage = (props) => {
 
 const mapStateToProps = state => {
   const loginPageState = state.login;
+  const thirdPartyAuthContext = thirdPartyAuthContextSelector(state);
```

Comment:
Lets put it directly in the return statement as it is only being used once. There is no need for a variable

Reply:
addressed